explorer/package.json

```diff
@@ -1,30 +1,28 @@
 {
-  "name": "explorer",
+  "name": "@chainlink/explorer",
   "private": true,
   "version": "0.0.0",
   "description": "LINK Explorer",
   "author": "Chainlink Dev Team",
   "license": "MIT",
   "keywords": [],
   "scripts": {
-    "dev": "yarn autosetup && concurrently \"cd client && yarn start\" \"ts-node-dev --respawn --transpileOnly ./src/index.ts\"",
+    "dev": "yarn automigrate && concurrently \"cd client && yarn start\" \"ts-node-dev --respawn --transpileOnly ./src/index.ts\"",
```

Comment:
Challenging to support `autosetup` within yarn workspace so I removed it. Open to thoughts on how we can achieve this is we still want it.

Reply:
Should be fine to just leave it in the base `package.json` and out of any of the sub projects?

Reply:
@j16r I stumbled across [npm pre & post hooks](https://docs.npmjs.com/misc/scripts#description) and used them instead.